modelforge/potential/utils.py

```diff
@@ -329,7 +329,7 @@ def _shifted_softplus(x: torch.Tensor):
     from torch.nn import functional
     import math
 
-    return functional.softplus(x) - math.log(0.2)
+    return functional.softplus(x) - math.log(2.0)
```

Comment:
Would it be better to keep everything in torch? 

Something like: 

`log_2 = torch.full((x.shape[0],), 2.0, device=x.device).log()
return functional.softplus(x)-log_2
`

Not sure if this would ultimately be more efficient.

Reply:
I believe that this computation is more difficult because it converts log_2 to a tensor, for which we need to keep track of the dimensions and the device (even though it is only a constant offset).
I have turned this in a nn.Module class, though, and calculate the offset once in the init using math, hopefully removing some of the overhead.